.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "semi": true,
+  "bracketSpacing": true,
+  "endOfLine": "lf"
+}
```

Comment:
I think that you are having problems getting master changes.. why do you have these changes.?

Reply:
removed.